File: Reforço de Lógica/Python/EX03.py
# ------------------- Verifica se números são divisíveis ------------------- #

dividendo = int(input("Digite o dividendo: ")) # Pedindo o dividendo da operação
divisor = int(input("Digite o disivor: ")) # Pedindo o divisor da operação

# Se o divisor for 0, avisamos que é impossível dividir por zero:
if divisor == 0:
    print("\033[;31mNão é possível dividir por 0. \033[m")
# Se a divisão resultar em 0 sobra, eles são divisíveis:
elif dividendo % divisor == 0:
    print(f"\033[;32mEsses números são divisíveis. O resultado dessa divisão é {int(dividendo/divisor)}.\033[m")
# Se não, só sobra a possibilidade dos números não serem divisíveis:
else:
    print("\033[;31mEsses números não são divisíveis.\033[m")

File: Reforço de Lógica/EX03.php
<?php 
// Recebendo os valores da divisão:
$dividendo = readline("Digite o dividendo: "); 
$divisor = readline("Digite o disivor: ");

if ($dividendo % $divisor == 0){ // Se a sobra da divisão for 0, então são divisíveis:
    echo "\033[;32mEsses números são divisíveis.\033[m";
} elseif ($divisor == 0){ // Se o divisor for 0, evitamos um erro ao lembrar que nenhum número é divisível por 0:
    echo "\033[;31mNão é possível dividir por 0. \033[m";
} else{ // Se não, eles não são divisíveis:
    echo "\033[;31mEsses números não são divisíveis.\033[m";
}
?>